src/config/deployed/mod.rs

```diff
@@ -88,6 +89,7 @@ pub fn get_configuration() -> Result<Configuration, NotifyServerError> {
         keypair_seed: config.keypair_seed,
         project_id: config.project_id,
         relay_url: config.relay_url,
+        relay_public_key: config.relay_public_key,
```

Comment:
There's an endpoint in the relay to fetch the key:
https://relay.walletconnect.com/public-key